searchcore/src/vespa/searchcore/proton/flushengine/flushengine.cpp

```diff
@@ -57,6 +57,12 @@ logTarget(const char * text, const FlushContext & ctx) {
         ctx.getHandler()->getCurrentSerialNumber());
 }
 
+bool
+reuse_strategy(const IFlushStrategy& old_strategy, const IFlushStrategy& strategy) {
+    return (old_strategy.name() == strategy.name() ||
```

Comment:
Would be nice with a comment about how `flush_all` subsumes `prepare_restart` and therefore means it can be reused